src/btree/bt_handle.c

```diff
@@ -626,6 +626,7 @@ __wt_btree_tree_open(WT_SESSION_IMPL *session, const uint8_t *addr, size_t addr_
     WT_BTREE *btree;
     WT_DECL_ITEM(tmp);
     WT_DECL_RET;
+    WT_DECL_STACK_ITEM(tmp_stack, 1024);
```

Comment:
These changes to two allocations in the btree code show how the allocator could be used in real code.  The choice of these places to change are not based on any performance data.